src/main/java/com/coreos/jetcd/EtcdWatch.java

```diff
@@ -0,0 +1,32 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.options.WatchOption;
+import com.coreos.jetcd.util.ListenableSetFuture;
+import com.coreos.jetcd.watch.Watcher;
+import com.google.protobuf.ByteString;
```

Comment:
IMHO, it's better to avoid exposing ByteString directly since Apache Felix's iPOJO abstraction can be broken. Let me explain the detail. protobuf compiler generates non-compatible source code(I mean it's source code-level compatibility, not wire-level compatibility). If a user of jetcd tries to use another protobuf compiler(e.g. v2.5.0 compiler) for her applications, it can happen for users to pass non-compatible(v2.5.0) ByteString bytecode. My suggestion is to create a wrapper class of ByteString for preventing users from touching jetcd's ByteString.